regression/abstraction/abstraction.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# whether verify the program or not
+CHECK=true
+
+# executables
+bindir="$(pwd)/../../build/bin"
+GOTO_CC="$bindir/goto-cc"
+GOTO_INSTRUMENT="$bindir/goto-instrument"
+CBMC="$bindir/cbmc"
+
+CBMC_FLAGS="--unwind 4 
```

Comment:
Why 4?

Reply:
This number comes from the particulars of our approach. For examples considered here, this unwinding number is sufficient with rr-abstraction.

Reply:
If it is specific to this example then that should be documented and explained.  If it is required for all uses of `--rr-abstraction` then maybe it could be set in the command-line argument parsing?